WebInstall/explainer_cross_domain.md

```diff
@@ -172,17 +172,37 @@ switch (state) {
     break;
 }
 ```
+<<<<<<< Updated upstream
 ####  **Install Sources manifest field**
 * A new field called `install_sources` will be added to the manifest file to have a control list of sites that can install the app. In its most restrictive case, the developer can specify to not allow installation from any other origin, in which case the PWA conforms to its usual behaviour of only being able to be installed from its same origin.
 
 ##### Allowing installations from any origin
 A new web-manifest boolean key `allow_all_install_sources` signals that the application can be installed from any source. If set to `true`, the `install_sources` list is ignored (if included). If set to `false` or absent, it defers to the origins listed in `install_sources`.
 
+=======
+####  **Controlling default installation sources**
+
+The default behaviour of a UA for the cross-origin Web Install API can be to allow installations from any origin or from no origin. *This default is defined by the implementer*. An implementer may choose to:
+
+* ALLOW cross-origin installations by default.
+* DENY cross-origin installations by default.
+
+This affects if an origin must be listed in the `install_sources` of an app to be able to install it.
+
+##### Overriding the default UA behaviour
+
+A developer can have full control of where their app can be installed from, independent of the implementor's default behaviour. A new web-manifest boolean key `allow_all_install_sources` can tell the UA that the application can be installed from any or no other origin. This overrides the default implementation by the UA.
+
+* if set to `true`, then cross-origin installations can enabled by default.
+* if set to `false`, only same-origin installations are allowed, unless the invoking installation-origin is listed in the application's `install_sources`.
```

Comment:
Should this also mention how it affects same-origin installation?

Reply:
these fields affect only cross-origin, same-origin wouldnt be affected, right?

Reply:
It'd still be good to call out that it explicitly it is only cross-origin and does _not_ affect same-origin, but that can be a follow up change, so I won't block on this PR.